package.json

```diff
@@ -15,5 +15,8 @@
     "eslint": "^7.13.0",
     "lerna": "^3.22.1",
     "prettier": "^2.2.0"
+  },
+  "dependencies": {
+    "eslint-plugin-node": "^11.1.0"
   }
```

Comment:
Remove this from the root's package.json. It should be in the `packages/eslint-config/package.json` file

Reply:
done